save-plugins/warn-plugin/src/commonNonJsTest/kotlin/org/cqfn/save/plugin/warn/WarnPluginTest.kt

```diff
@@ -1,479 +0,0 @@
-package org.cqfn.save.plugin.warn
```

Comment:
There is no place for test in our project. Please avoid creating such things next time. Completely useless

Reply:
Why?

Reply:
fixed